tests/v2/validation/rbac/rbac_test.go

```diff
@@ -365,22 +370,26 @@ func (rb *RBTestSuite) TestRBAC() {
 		})
 
 		//Verify standard users cannot list any clusters
-		rb.Run("Test case Validate standard users cannot list any downstream clusters before adding the cluster role "+tt.name, func() {
+		rb.Run("Test case Validate if users can list any downstream clusters before adding the cluster role "+tt.name, func() {
 			_, err := rb.standardUserClient.Steve.SteveType(clusters.ProvisioningSteveResouceType).ListAll(nil)
-			require.Error(rb.T(), err)
-			assert.Equal(rb.T(), "Resource type [provisioning.cattle.io.cluster] is not listable", err.Error())
+			if tt.role == standardUser{
```

Comment:
Nit, the spacing seems a bit different than what it is supposed to be, could you run go fmt

Reply:
I came from the perspective where Standard user/Admin/Restricted admin is a global role. So, I made standarduser and restricted admin as role and since the roles project owners/cluster owners etc are the members of a cluster or project . 
But I can understand the confusion here and updated it to role == clusterOwner, projectOwner etc and member == standard user and restricted admin.